README.md

```diff
@@ -1,43 +1,42 @@
-# repo-report <sup>[![Version Badge][npm-version-svg]][package-url]</sup>
+# 🧾 Repo-Report
 
-[![github actions][actions-image]][actions-url]
-[![coverage][codecov-image]][codecov-url]
-[![dependency status][deps-svg]][deps-url]
-[![dev dependency status][dev-deps-svg]][dev-deps-url]
-[![License][license-image]][license-url]
-[![Downloads][downloads-image]][downloads-url]
+**Repo-Report** is a simple command-line tool (CLI) that helps you get a quick overview of all your GitHub repositories in one place.
 
-[![npm badge][npm-badge-png]][package-url]
+Instead of clicking through every repo manually on GitHub, Repo-Report shows important info like whether issues are turned on, if the repo has a license, when it was last updated, and more — all from one command.
 
-CLI to list all repos a user has access to, and report on their configuration in aggregate.
+It’s perfect for developers, students, or maintainers who want a clear summary of their GitHub projects.
 
-# Installation
+---
 
-- `npm install` to install all dependencies
-- create `.env` file and initialize `GH_TOKEN` or `GITHUB_TOKEN` (in order of precedence) with your Github token
+## 🛠️ How It Works
 
-# Usage (for public)
+1. **You run the tool in your terminal.**
+2. **Repo-Report connects to GitHub using your personal access token (PAT).**  
+   This is a password-like key GitHub gives you to allow tools to access your repos.
+3. **It scans every repository you have access to** — personal or organizational.
+4. **You get a report in your terminal** that shows useful details about each repo.
 
-- generate a personal access token using github [here](https://github.com/settings/tokens) and add the `repo` scope to it.
-- on the terminal run `export GH_TOKEN=<the personal access token generated>`
-- run `npx repo-report`
```

Comment:
are you sure you want to delete this information?

how do I know where to put my GH_TOKEN now?